data_aquisition.py

```diff
@@ -52,111 +54,131 @@
-            
-            ##Excluding videos present in both EM and non-EM
-            if(set(v_class)&set(em_class.keys())!=set([])):
-                flg1=1
-            if(set(v_class)&set(non_em_class.keys())!=set([])):
-                flg2=1
-
-            if flg1==1 and flg2==1:
-                continue;
-            elif flg1==1:
-                print (v_id,v_start,v_end,'em')
-                ydl_opts = {
-                        'format': 'bestaudio/best',
-                    'ignoreerrors':'True',
-                        'postprocessors': [{
-                            'key': 'FFmpegExtractAudio',
-                            'preferredcodec': 'wav',
-                            'preferredquality': '192',
-                        }],
-                        'outtmpl':'emergency/%(id)s.%(ext)s',    
-                }
-
-                with youtube_dl.YoutubeDL(ydl_opts) as ydl:
-                        ydl.download(['http://www.youtube.com/watch?v='+row[0]])
-
-                input_file = em_download_path+v_id+'.wav'
-                print(input_file)
-                # break
-                if(os.path.exists(input_file)):
-                    print ('#############################################')
-                    em_c = em_c +1
-                    output_file = em_files_path+str(em_c)+'.wav'
-                    print (output_file)
-                    ffmpeg_extract_subclip(input_file,v_start,v_end,targetname=output_file)
-                    os.remove(input_file)
-                    
-            elif flg2==1:
-                print (v_id,v_start,v_end,'Non-em')
-                ydl_opts = {
-                        'format': 'bestaudio/best',
-                    'ignoreerrors':'True',
-                        'postprocessors': [{
-                            'key': 'FFmpegExtractAudio',
-                            'preferredcodec': 'wav',
-                            'preferredquality': '192',
-                        }],
-                        'outtmpl':'nonEmergency/%(id)s.%(ext)s',    
-                }
-
-                with youtube_dl.YoutubeDL(ydl_opts) as ydl:
-                        ydl.download(['http://www.youtube.com/watch?v='+row[0]])
+
+def prepare_data(args):
+    '''
+    Prepares data in the 2 created sub-folders, 'emergency' and 'nonEmergency'
+    '''
+    base_download_path = str(args.download_dir)
+
+    em_download_path = os.path.join(base_download_path, 'emergency')
+    nonem_download_path = os.path.join(base_download_path, 'nonEmergency')
+
+    base_save_path = str(args.save_dir)
+
+    em_files_path = os.path.join(base_save_path, 'emergency')
+    nonem_files_path = os.path.join(base_save_path, 'nonEmergency')
+
+    ##################################
+    os.chdir(base_download_path)
+
+    # mention the name of the .csv file
+    with open('eval_segments.csv') as csvfile:   # Also use other csv audioset files too extract all the data available in these categories
```

Comment:
Can you make an argument for the .csv file as well? There are two csv files that need to be downloaded eval and balanced. The idea is to take the csv file name from the user and then prepare the data in that csv in the given folders.

Reply:
Also please update the README for instructions on script usage

Reply:
I'll make the changes :+1: